### PR TITLE
Fixed error statments

### PR DESCRIPTION
**Pull request by nathannaveen**
- Changed the errors from returning `return err` to `return fmt.Errorf()`.
- Only changed some of the errors because it would be too big of the PR otherwise and would take forever.
- Helps with https://github.com/guacsec/guac/pull/423#discussion_r1102924221

Signed-off-by: nathannaveen <42319948+nathannaveen@users.noreply.github.com>

### CODE REVIEW

File: cmd/guacone/cmd/files.go [hunk 1]
```diff
@@ -207,14 +207,14 @@ func validateFlags(user string, pass string, dbAddr string, realm string, keyPat
...
-		return process.Process(ctx, d)
+		return process.Process(ctx, d) // nolint:wrapcheck
 	}, nil
 }
 func getIngestor(ctx context.Context) (func(processor.DocumentTree) ([]assembler.Graph, error), error) {
 	return func(doc processor.DocumentTree) ([]assembler.Graph, error) {
 		inputs, err := parser.ParseDocumentTree(ctx, doc)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("unable to parse document tree: %v", err)
```
Comment (lumjjb) [inline cmd/guacone/cmd/files.go:217]: probably should be %w for wrapping error

**Review by lumjjb — CHANGES_REQUESTED** [review]
there seem to be a couple where the format string uses %v or %s instead of %w. can you change that please. Thanks!

File: cmd/guacone/cmd/files.go (hunk 1 above)
Comment (nathannaveen) [reply cmd/guacone/cmd/files.go]: Good idea, I forgot about that. I have changed all the `%v`’s that I have added to `%w`’s.

**Comment by nathannaveen** [comment]
Closing this PR in favor of doing smaller versions of this because I am unable to debug where the integration tests are failing.

**Pull request closed without merging**